contracts/PBAB+Collabs/9dcc-iyk/DEPLOYMENTS.md

```diff
@@ -0,0 +1,17 @@
+# Deployments Log
+
+## Mainnet
+
+**GenArt721CoreV2_ENGINE_FLEX:** https://etherscan.io/address/TODO#code
+**GenArt721Minter:** https://etherscan.io/address/TODO#code
+
+`TODO` set as minter owner, whitelisted on core contract, and admin of core contract.
+
+## Goerli
+
+### Upload 0
+
+**GenArt721CoreV2_ENGINE_FLEX:** https://goerli.etherscan.io/address/0xD94C7060808f3c876824E57e685702f3834D2e13#code
+**GenArt721Minter:** https://goerli.etherscan.io/address/0x5FE770CF8Ad1DF5DDe2c1447A898B087c275F6c1#code
+
+`TODO` set as minter owner, whitelisted on core contract, and admin of core contract.
```

Comment:
does this need to be updated with whatever was done during deployment?

Reply:
Good catch, yes! Updated (to `0x5FE770CF8Ad1DF5DDe2c1447A898B087c275F6c1`).